### PR TITLE
Feature/print seating chart

### PR DESCRIPTION
**Pull request by dee-dee-raye**
To test:
got to - http://127.0.0.1:5000/printSeatingChartTest
plug in an event id. It should download that event's seating chart. If the event doesn't have any guests those fields are empty.

### CODE REVIEW

**Comment by dee-dee-raye** [comment]
@epielemeier 
@jake6642 
@pnoorossana 

Chance helped me get this working. It's good to review again.

File: eagleEvents/printing/chart.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import sys
 import os
 import datetime
+from io import BytesIO
```
Comment (simpleman19) [inline eagleEvents/printing/chart.py]: Can you clean up some of your unused imports?

File: eagleEvents/templates/print.html.j2 [hunk 2]
```diff
@@ -9,7 +9,7 @@
         <form id="seating-chart">
```
Comment (simpleman19) [inline eagleEvents/templates/print.html.j2]: You can probably take this out of the form since it is all over an ajax request

**Comment by epielemeier** [comment]
@dee-dee-raye Can you take care of this branch's merge conflicts?

**Pull request merged**